generators/spa/templates/Vue3JS/Module.dnn

```diff
@@ -0,0 +1,101 @@
+<dotnetnuke type="Package" version="5.0">
+  <packages>
+    <package name="<%= fullNamespace %>" type="Module" version="01.00.00">
```

Comment:
@alejoroman0605 There is a token for this version number.  https://github.com/UpendoVentures/generator-upendodnn/blob/master/generators/spa/index.js#L121

Reply:
@WillStrohl I just fixed it https://github.com/UpendoVentures/generator-upendodnn/pull/280/commits/c0437c2115848e91e47ac709f478d2f74ff60337